### PR TITLE
Instructor Session page: reminder dropdown button is not disabled for instructor who does not have permission for it

### PR DESCRIPTION
**Pull request by franklingu**
for #2848

### CODE REVIEW

**Comment by damithc** [comment]
Look through the changes to html files to ensure they are relevant changes or harmless formatting changes. Can merge after that.

**Comment by franklingu** [comment]
I asked @LowWeiLin to run on this machine and things are fine

**Comment by damithc** [comment]
Changes to html files means we have changed the expected behavior of some
tests. Usually we go through the those changes in html files manually to
ensure those changes are the right changes. But it is not possible to do
that in this case because there are massive unrelated changes. While the
tests still pass, we don't know whether the tests are correct anymore.

**Comment by franklingu** [comment]
@LowWeiLin can you just run those 2 tests for me on this branch?
"instructorHomePage is failing for me, and that cause adminHomepage to fail also"

**Comment by franklingu** [comment]
@damithc , ready to merge?

**Comment by damithc** [comment]
I couldn't find the change in html files that is relevant to this fix. Can you point it out?

File: src/main/java/teammates/ui/controller/PageData.java [hunk 1]
```diff
@@ -894,8 +894,8 @@ public String getInstructorFeedbackSessionActions(FeedbackSessionAttributes sess
             "title=\"" + Const.Tooltips.FEEDBACK_SESSION_REMIND + "\" data-toggle=\"tooltip\" data-placement=\"top\"" +
             (hasRemind ? "onclick=\"return toggleRemindStudents('" + session.feedbackSessionName + "');\" " : "") +
             disableRemindSessionStr + ">Remind</a>" +
-            "<button type=\"button\" class=\"btn btn-default btn-xs btn-tm-actions dropdown-toggle session-remind-options-for-test" + 
-            (hasRemind ? "\"" : DISABLED) +
+            "<button type=\"button\" class=\"btn btn-default btn-xs btn-tm-actions dropdown-toggle session-remind-options-for-test\"" + 
+            disableRemindSessionStr +
```
Comment (franklingu) [inline src/main/java/teammates/ui/controller/PageData.java]: this is the change. the related HTML is WithHelperView

File: src/test/resources/pages/instructorFeedbacksAllSessionTypesWithHelperView.html [hunk 2]
```diff
@@ -467,7 +467,7 @@ <h5 class="col-sm-2">
...
...="false"> <span class="caret"></span></button> <ul class="dropdown-menu" role="menu"><li><a href="/page/instructorFeedbackRemind?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" class="session-remind-inner-for-test" title="Send e-mails to remind students and instructors who have not submitted their feedbacks to do so" data-toggle="tooltip" data-placement="top">Remind all students</a></li><li><a href="#" data-actionlink="/page/instructorFeedbackRemindParticularStudentsPage?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" class="session-remind-particular-for-test" data-courseid="CFeedbackUiT.CS1101" data-fsname="Copied Session" data-toggle="modal" data-target="#remindModal">Remind particular students</a></li></ul></div> <a class="btn btn-default btn-xs btn-tm-actions session-publish-for-test disabled" onclick="return false" href="/page/instructorFeedbackPublish?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;next=%2Fpage%2FinstructorFeedbacksPage%3Fuser%3DCFeedbackUiT.helper&amp;user=CFeedbackUiT.helper" title="This session is not yet opened" data-toggle="tooltip" data-placement="top">Publish</a> </td>
```
Comment (franklingu) [inline src/test/resources/pages/instructorFeedbacksAllSessionTypesWithHelperView.html]: this line is different. remove the disabled for caret. adding disabled for links in drop down instead

**Comment by damithc** [comment]
Go ahead and merge

**Pull request merged**